### DISCUSSION TITLE
关于VMware中部署的doris升级

### DISCUSSION THREAD

**Discussion by 0xcca** [apache/doris#20229] [A - General / Q&A]
背景：
因为正式环境很早就部署了doris0.15.0，奈何版本实在太低很多功能都不支持，故打算先升级到1.1.5，再升级到最新版1.2.4.1，考虑到对升级操作不熟悉想在虚拟环境练练手，结果出现了一些问题。

问题：
我在VMware虚拟机中的Linux7系中部署了doris0.15.0，be升级很顺畅没有问题，但升级fe却无法获取ip，而且我的配置文件还指定了priority_networks，请求支援！！！

<img width="987" alt="WechatIMG108" src="https://github.com/apache/doris/assets/51154276/44f47d23-2ea9-4815-ac43-080b4fa4605b">

**Comment by LemonLiTree** [comment]
set metadata_failure_recovery = true in fe.conf , then restart fe

**Comment by 0xcca** [comment]
后面排查后发现，是docker影响了fe读取ip，无论我指定了ip从/16/24/32都会读取不到ip，我换了一个没有docker的环境就可以正常升级。
